bookkeeper-server/src/test/java/org/apache/bookkeeper/client/ExplicitLacTest.java

```diff
@@ -50,6 +53,12 @@ public ExplicitLacTest(Class<? extends LedgerStorage> storageClass) {
         super(1);
         this.digestType = DigestType.CRC32;
         baseConf.setLedgerStorageClass(storageClass.getName());
+        /*
+         * to persist explicitLac, journalFormatVersionToWrite should be atleast
+         * V6 and fileInfoFormatVersionToWrite should be atleast V1
+         */
+        baseConf.setJournalFormatVersionToWrite(6);
```

Comment:
@reddycharan maybe it is better to use defined constants for these values (I mean '6' here and '1' below)

Reply:
yes, I thought about this. But "JournalChannel" is not public class (which contains defined constants) and similarly "FileInfo" is not public either. 

I guess for the same reason actual values are used in other places aswell - https://github.com/apache/bookkeeper/blob/master/bookkeeper-server/src/test/java/org/apache/bookkeeper/conf/TestBKConfiguration.java#L50

https://github.com/apache/bookkeeper/blob/master/bookkeeper-server/src/main/java/org/apache/bookkeeper/conf/ServerConfiguration.java#L637

Reply:
Oh, ok!
Thank you for your explanation

Reply:
Oh, ok!
Thank you for your explanation